packages/lib-utils/package.json

```diff
@@ -10,7 +10,9 @@
   },
   "files": [
     "dist/index.js",
-    "dist/index.d.ts"
+    "dist/index.d.ts",
+    "dist/index.css",
```

Comment:
@florkbr @fhlavac This should fix the issue of missing CSS :smiley: 

It's generated, but it wasn't part of `files` and therefore wasn't published.